test/test_mps.py

```diff
@@ -4736,6 +4736,14 @@ def helper(n, m, dtype):
             helper(3, 8, dtype)
             helper(8, 3, dtype)
 
+    # Test argsort
+    def test_argsort(self):
+        def helper(stable, dim, descending):
+            argsort_mps_result = torch.argsort(stable, dim, descending).to("mps")
+            argsort_cpu_result = torch.argsort(stable, dim, descending).to("cpu")
+
+            self.assertEqual(argsort_mps_result, argsort_cpu_result)
+
```

Comment:
call the helper function

Reply:
Added sentences to call the helper function. Is my understanding correct?

https://github.com/pytorch/pytorch/pull/87572/commits/eae816354d1973b14ab59d38016dc6fc54a4cb1a